api/src/pcapi/core/finance/api.py

```diff
@@ -1494,11 +1501,72 @@ def _mark_as_processed(pricings: sqla_orm.Query) -> None:
+                    .join(models.FinanceEvent.pricings)
+                    .outerjoin(models.Pricing.cashflows)
+                    .filter(
+                        models.FinanceIncident.venueId.in_(venue_ids),
+                        models.FinanceIncident.status == models.IncidentStatus.VALIDATED,
+                        models.Cashflow.id.is_(None),  # Add this line to filter incidents with cashflows
+                    )
+                    .all()
+                )
+
+                if not total or (total > 0):
+                    last_cashflow_age = 0
+
+                    if total > 0:
+                        override_incident_debit_note = (
+                            models.FinanceIncident.query.join(models.FinanceIncident.booking_finance_incidents)
+                            .join(models.BookingFinanceIncident.finance_events)
+                            .join(models.FinanceEvent.pricings)
+                            .outerjoin(models.Pricing.cashflows)
+                            .filter(
+                                models.FinanceIncident.venueId.in_(venue_ids),
+                                models.FinanceIncident.status == models.IncidentStatus.VALIDATED,
+                                models.FinanceIncident.forceDebitNote == True,
+                                models.Cashflow.id.is_(None),  # Add this line to filter incidents with cashflows
+                            )
+                            .one_or_none()
+                        )
+
+                        last_cashflow = (
+                            models.Cashflow.query.filter(
+                                models.Cashflow.reimbursementPointId == reimbursement_point_id,
+                                models.Cashflow.bankAccountId == bank_account_id,
+                                models.Cashflow.status == models.CashflowStatus.PENDING,
+                            )
+                            .order_by(models.Cashflow.creationDate.desc())
+                            .one_or_none()
+                        )
+                        if last_cashflow:
+                            last_cashflow_age = (datetime.datetime.utcnow() - last_cashflow.creationDate).days
+
+                        if (
+                            last_cashflow_age < conf.DEBIT_NOTE_AGE_THRESHOLD_FOR_CASHFLOW
+                            or not override_incident_debit_note
+                        ):
+                            for incident in all_current_incidents:
+                                history_api.log_action(
+                                    history_models.ActionType.FINANCE_INCIDENT_WAIT_FOR_PAYMENT,
+                                    author=None,
+                                    finance_incident=incident,
+                                    comment="Le montant de l’incident est supérieur au montant total des réservations validée de y euros donc aucun justificatif n’est généré, on attend la prochaine échéance",
```

Comment:
```suggestion
                                    comment="Le montant de l’incident est supérieur au montant total des réservations validées de y € donc aucun justificatif n’est généré, on attend la prochaine échéance",
```
Deux commentaires supplémentaires
- la valeur de `y` serait bienvenue !
- le message semble en partie redondant avec le texte de l'`actionType`